db/seeds.rb

```diff
@@ -7,98 +7,99 @@
 
 puts "Seed Start"
```

Comment:
Let's not forget to NOT seed or destroy the Works in production, and to play the Bibliography csv in production just Once. 
💪👾

Reply:
Everything is in developpement only, just the exhibition for the moment, and we can easily destroy it :)

Reply:
don't be affraid ! :p